elasticsearch/Dockerfile

```diff
@@ -8,10 +8,10 @@ USER 0
 
 ENV HOME=/opt/app-root/src \
   JAVA_VER=1.8.0 \
-  ES_VER=2.4.1 \
-  SG_VER=2.4.1.8 \
-  SG_SSL_VER=2.4.1.16 \
-  ES_CLOUD_K8S_VER=2.4.1 \
+  ES_VER=2.4.4 \
+  SG_VER=2.4.4.10 \
+  SG_SSL_VER=2.4.4.19 \
+  ES_CLOUD_K8S_VER=2.4.4 \
   OSE_ES_VER=2.4.1.2 \
```

Comment:
I'm surprised that it installs then... the plugins specify the version of ES that they need to work with (big pain with the 2.x ES plugins)